src/lib/db/indexedDBService.js

```
@@ -206,197 +343,235 @@ async function getHistoryById(id) {
         await addSummary(summaryToArchive)
-        // Cập nhật trạng thái isArchived của mục lịch sử gốc
         await updateHistoryArchivedStatus(historyId, true)
         resolve(true)
       } else {
         reject('History item not found.')
       }
     } catch (error) {
-      console.error('Error moving history item to archive:', error)
       reject(error)
     }
   })
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)